q2_diversity_lib/tests/test_alpha.py

```diff
@@ -32,56 +30,89 @@ def setUp(self):
 
     def test_non_phylogenetic_passed_empty_table(self):
         for measure in nonphylogenetic_measures:
-            with self.assertRaisesRegex(ValueError, "empty"):
+            with self.assertRaisesRegex(ValueError, 'empty'):
                 measure(table=self.empty_table)
 
 
 class FaithPDTests(TestPluginBase):
-
     package = 'q2_diversity_lib.tests'
```

Comment:
I know this isn't being added in this PR, but you should fix it here - the `package` attr is "The name of the plugin package to be tested." (https://github.com/qiime2/qiime2/blob/57827c83b39823f497c5f05454cd605edbb77d95/qiime2/plugin/testing.py#L31). Please drop the `.tests` from the end. Also, please ensure that this change is propagated to any other test suites that use the `TestPluginBase` harness.

Reply:
This change causes `FileNotFoundError`s when [`get_data_path`](https://github.com/qiime2/qiime2/blob/57827c83b39823f497c5f05454cd605edbb77d95/qiime2/plugin/testing.py#L100) goes looking for files in `~/.../q2_diversity_lib/data` instead of `~/.../q2_diversity_lib/tests/data`. 

This approach (`<package>.tests`) is used in [`q2-types`](https://github.com/qiime2/q2-types/blob/3836562ced8f1875de654c956f8161f10b592a11/q2_types/feature_data/tests/test_format.py#L26) (which is linked from the dev docs) and `q2-cutadapt`, but there are other options. I can bump the `data` directory up out of `tests`, which also does funny things to the semantics, and I'm sure there are larger-scale changes we could make to avoid `get_data_path` locally, modify it, and probably others that aren't jumping to mind. 

I'm inclined to let sleeping dogs lie. What are your thoughts?

Reply:
Oops, my mistake, yeah, this is fine. The plugin's package is addressed in the test harness here: https://github.com/qiime2/qiime2/blob/57827c83b39823f497c5f05454cd605edbb77d95/qiime2/plugin/testing.py#L50, which means this is right as-is. Nevermind me...